client/src/pages/feedback.tsx

```diff
@@ -116,6 +116,17 @@ const columnHeaders: ColumnDef[] = [
   },
 ];
 
+function GetCompleteAnswers(mentorAnswers: Answer[]) {
```

Comment:
This works totally fine, but this whole function can be achieved in a single line using the filter function. If you have the time, give it a shot. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter